doc/modules/cross_validation.rst

```diff
@@ -548,6 +574,25 @@ to shuffle the data indices before splitting them. Note that:
 * To ensure results are repeatable (*on the same platform*), use a fixed value
   for ``random_state``.
 
+Cross validation of time series data
+====================================
+
+Time series data is characterised by the correlation between observations 
+that are near in time (*autocorrelation*). However, classical 
+cross-validation techniques such as :class:`KFold` and 
+:class:`ShuffleSplit` assume the samples are independent and 
+identically distributed, and would result in unreasonable correlation 
+between training and testing instances (yielding poor estimates of 
+generalisation error) on time series data. Therefore, it is very important 
+to evaluate our model for time series data on the "future" observations 
+least like those that are used to train the model. To achieve this, we 
+need to adopt the special time series cross-validation which first split 
+the data into :math:`n\_splits + 1` folds, then in each iteration 
+:math:`i`, use first :math:`i` fold(s) as training set, and use 
+:math:`i+1` th fold as testing set.
+:class:`HomogeneousTimeSeriesCV` implements the cross-validation 
+technique described above.
```

Comment:
How can I insert a link on "above"?

I can only link to the anchor by typing `homogeneous_time_series_cv_` in .rst file, which will be rendered as `homogeneous_time_series_cv` instead of `above`.